### PR TITLE
Update decentlab devices

### PR DESCRIPTION
**Pull request by decentlab**
#### Summary

Adds devices and photos

#### Changes

- added dl-ntu
- added dl-lws
- added dl-itst-002
- added dl-wrm-002
- added additional photos for other devices
- tweaked vendor info links

#### Notes for Reviewers

- added two new sensors `leaf wetness` and `turbidity`

### CODE REVIEW

File: lib/draft/schema.json [hunk 1]
```diff
@@ -958,15 +960,7 @@
...
-            "enum": [
-              "modbus",
-              "m-bus",
-              "can bus",
-              "rs-485",
-              "sdi-12",
-              "analog",
-              "dali"
-            ]
+            "enum": ["modbus", "m-bus", "can bus", "rs-485", "sdi-12", "analog"]
```
Comment (Jaime-Trinidad) [inline lib/draft/schema.json:963]: please leave them in a list form for order, and `dali` is missing

**Comment by Jaime-Trinidad** [comment]
@decentlab thanks for the updates, there is one comment and please make a second check in the images, the ones that have background aren't the `main` images, main images should be without background for the device page.

File: lib/draft/schema.json (hunk 1 above)
Comment (decentlab) [reply lib/draft/schema.json]: Fixed the mistake.

NB: also added downlink codecs.

**Comment by decentlab** [comment]
@Jaime-Trinidad, thanks for the review.

- fixed the lib/draft merge mistake
- checked all main photos for PNG extension.
- added downlink codecs.

**Pull request merged**